src/components/pages/Flagging/FlagFeed.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * The Feed is a great place for members to catch up on what's happened recently!
+ * Shows all of the most recent raha gives and verified join videos.
```

Comment:
I don't think this comment is correct?

Reply:
Oh yeah, copy pasta'd from some `Feed.tsx` lol.